src/index.html

```diff
@@ -17,6 +17,14 @@
     />
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="container">
+      <div class="stop-watch stop-watch__speed-up--10s">
```

Comment:
The class 'stop-watch__speed-up--10s' seems to be a modifier class. According to BEM methodology, it should be used with a block or element class. Ensure that it is correctly applied as a modifier.